src/app/presentation/arrows/arrows.component.ts

```diff
@@ -1,5 +1,5 @@
-import {Component, OnInit} from '@angular/core';
-import {PresentationComponent} from '../presentation/presentation.component';
+import { Component, OnInit } from '@angular/core';
```

Comment:
@kirjs tslint.json has import spacing = true. Is your IDE set up to use tslint.json?